Source/SharpNeedle/Glitter/Effect.cs

```diff
@@ -0,0 +1,35 @@
+﻿using SharpNeedle.BINA;
+
+namespace SharpNeedle.Glitter;
+
+[NeedleResource("glitter/effect", @"\.(gle|effect)$")]
+public class Effect : BinaryResource
+{
+    public new static readonly uint Signature = BinaryHelper.MakeSignature<uint>("FIRG");
+    public uint Version { get; set; }
```

Comment:
Consider adding a default assignment to this. In case of manual construction you wouldn't want to set the version field manually.

Reply:
Fixed in 1ee34e0.